lib/src/validator/size.dart

```diff
@@ -8,6 +8,9 @@ import 'dart:math' as math;
 import '../entrypoint.dart';
 import '../validator.dart';
 
+import 'package:path/path.dart';
+import 'package:pub/src/io.dart';
```

Comment:
It's best to import files within the same `lib/` directory using relative imports. Otherwise you can run into canonicalization issues where the Dart VM ends up loading the same file twice.

Reply:
Sure.